Gemfile.lock

```diff
@@ -230,9 +230,9 @@ GEM
     pundit (2.1.0)
       activesupport (>= 3.0.0)
     racc (1.6.0)
-    racecar (2.2.0)
+    racecar (2.4.0)
```

Comment:
* Handle ERR_NOT_COORDINATOR (#209)
* Update librdkafka version from 1.4.0 to 1.5.0 by upgrading from rdkafka 0.8.0 to 0.10.0. (#263)
* Restore support for Ruby 2.4 (#258)